Adds many new species and their respective armour / items.

X0-11 opened the pull request:
Species added:
Spartans
Jackals
Brutes (TODO)
Species icons modified:
Sangheili

Will probably have merge conflicts if /when pr #295 is merged.

X0-11 committed:
Merge pull request #1 from HaloSpaceStation/alpha

a

X0-11 committed:
Merge pull request #2 from HaloSpaceStation/alpha

update

X0-11 committed:
Merge pull request #3 from HaloSpaceStation/alpha

update

X0-11 committed:
Merge branch 'alpha' of https://github.com/X0-11/HaloSpaceStation13 into alpha

X0-11 committed:
Merge branch 'alpha' of https://github.com/X0-11/HaloSpaceStation13 into alpha

X0-11 committed:
Merge branch 'alpha' of https://github.com/X0-11/HaloSpaceStation13 into alpha

X0-11 committed:
most stuff done

Massive error: Icons load wrong

bloxgate commented on code/modules/halo/weapons/covenant/ammo.dm:
```diff
@@ -6,11 +6,12 @@
-	icon = 'code/modules/halo/icons/Covenant Weapons.dmi'
-	icon_state = "PlasmapistolProj"
+	icon = 'code/modules/halo/icons/Covenant_Projectiles.dmi'
+	icon_state = "Plasmapistol Shot"
```
Plasma pistol is two words

bloxgate commented on code/modules/halo/weapons/covenant/ammo.dm:
```diff
@@ -19,8 +20,8 @@
-	icon = 'code/modules/halo/icons/Covenant Weapons.dmi'
-	icon_state = "PlasmarifleProj"
+	icon = 'code/modules/halo/icons/Covenant_Projectiles.dmi'
+	icon_state = "Plasmarifle Shot"
```
Two words

TheScreamingStar commented:
Later on I will finish up the rest on I will finish on the rest of the stuff for my sprites on this PR (higher rank armor, furred brutes, species aesthetics for them).

X0-11 committed:
update

Augmented limbs for spartans.
Kig-Yar helmet w/ nightvision.
Kig-Yar innate zoom.

bloxgate commented on code/modules/halo/species_items/kig-yar.dm:
```diff
@@ -5,6 +5,51 @@
+	var/linked_glasses
+	var/mob/living/carbon/human/u
+
+/obj/item/clothing/head/helmet/kigyar/proc/tryequipNV()
```
Could you put some underscores between the words in that proc name please?

bloxgate commented on code/modules/mob/living/carbon/human/species/outsider/brutes.dm:
```diff
@@ -0,0 +1,15 @@
+	blurb = ""//Not in chargen
+	flesh_color = "#4A4A64"
+	blood_color = "#A10808"
+	icobase = 'code/modules/halo/icons/species/r_Augmented_Human.dmi' //The DMI needed modification to fit the usual format (see other species' dmis)
```
This is temporary, I hope?

X0-11 replied:
Yep. My copypasta for species has been revealed!

bloxgate reviewed (changes requested):
Looking great so far XO! Just a couple questions though. Shouldn't the other covie species have some radiation resistance like the elites too? They are going to be using the same weapons after all.

Also, what does setting the species eyes to null do?

X0-11 commented:
eyes = null stops the species sprite assembler from drawing an eye overlay. Don't have one for those species yet.

X0-11 committed:
small changes

caelaislinn commented:
There's a couple of .tmp files there I'm not sure about, but this branch is getting deprecated shortly so we can just skip them when we come back to port them over.

caelaislinn commented:
Manual merging this...
